### DISCUSSION TITLE
Updated star trek list

### DISCUSSION THREAD

**Discussion by JasonMFults** [meisnate12/Plex-Meta-Manager#1881] [Show and tell]
I noticed that the list used for the Star Trek playlist hadn't been updated for almost a year and was missing the newest seasons of 3 shows and having some of the episodes out of order so I made a current/fixed list. https://trakt.tv/lists/27085572

**Comment by YozoraXCII** [comment]
https://github.com/meisnate12/Plex-Meta-Manager/pull/1894
